docs/usage/configuration-options.md

```diff
@@ -1852,6 +1852,11 @@ It will be compiled using `handlebars` and the regex `groups` result.
 If the `versioning` for a dependency is not captured with a named group then it can be defined in config using this field.
 It will be compiled using `handlebars` and the regex `groups` result.
 
+### registryUrlTemplate
+
+If the `registryUrls` for a dependency is not captured with a named group then it can be defined in config using this field.
+It will be compiled using `handlebars` and the regex `groups` result.
```

Comment:
```suggestion
It will be compiled using Handlebars and the regex `groups` result.
```

Would this be better? Or are we referring to a `handlebars` function name here?

Reply:
In this scenario it is referring to the templating library, rather than a specific function name. I copied this line of documentation from the four template descriptions above, happy to update them all to the suggestion if desired

Reply:
Ah you're right the previous lines all use `handlebars` as well.

@rarkins Which do you prefer:
`handlebars` or Handlebars?

Reply:
Either is fine, but let's be consistent

Reply:
I'll take "Handlebars" then, and let's be consistent with it. 😄